_posts/en/newsletters/2019-09-11-newsletter.md

```diff
@@ -0,0 +1,180 @@
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[LND][lnd repo], [C-Lightning][c-lightning repo], [Eclair][eclair repo],
+[libsecp256k1][libsecp256k1 repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #16185][] updates the `gettransaction` RPC with a new parameter
+  `decode`.  If set to True, a `decoded` field will be added to the RPC
+  output containing a version of the transaction decoded into JSON fields
+  (the same format used when requesting verbose output with the
+  `getrawtransaction` RPC).
+
+- [Bitcoin Core #15759][] increases the number of outbound connections
+  the node will make from 8 to 10.  The two new connections will only be
+  used to announce and relay new blocks; they won't announce or relay
+  unconfirmed transactions.  Only announcing blocks minimizes the
+  bandwidth overhead of the new connections and doesn't give spy nodes
+  information about what node first announced a transaction (i.e. which
+  node may have created it).  These extra two connections increases the
+  density of connections between network nodes, making it more difficult
+  to partition the network and execute various attacks possible against
+  isolated nodes.  If no problems are found with this change and the
+  bandwidth overhead remains small, it's possible the number of
+  blocks-only peers may be increased in later versions, either as its
+  own change or in conjunction with other ongoing development such as
+  [Erlay][].
+
+- [Bitcoin Core #14540][] allows users to create new wallets for
+  multiwallet mode from the GUI, completing a set of GUI actions that
+  also allows users to load and unload wallets.  The action opens a
+  dialog that lets the user name the wallet and set various wallet
+  options.
+
+- [Bitcoin Core #16421][] follows up [PR#15681][Bitcoin Core #15681]
+  (described in [Newsletter #56][carve-out]) by allowing a carve-out
+  transaction to be replaced using RBF.  Carve-out transactions are
+  allowed to slightly exceed Bitcoin Core's limits on transaction size
+  and number of ancestors.  When carve-out was added, the exception to
+  those rules was not applied to transaction replacements, so nodes
+  would accept carve-outs but not RBF fee bumps of them.  With this PR,
+  RBF fee bumping of the carve-out is now possible, making it an even
+  more useful tool for fee management of settlement transactions in
+  two-party LN payment channels.
+
+- [LND #3401][] caps the amount of onchain fee that a node will propose
+  paying in a channel update transaction (commitment transaction),
+  limiting it to 50% of the node's current in-channel balance (the 50%
+  default is adjustable).  In theory, the channel can still be used
+  after this---which is why it isn't closed---although the node may not
+  have enough funds to initiate a spend, possibly making receiving its
+  only option unless onchain feerates drop.  Because of a limitation of
```

Comment:
perhaps s/onchain/on-chain/ and s/Because/Due to/

Reply:
https://github.com/bitcoinops/bitcoinops.github.io/blob/master/STYLE.md#compound-words says "onchain"